extensions/punto/src/index.ts

```diff
@@ -66,11 +73,11 @@ function detectLayout(input: string): string {
 function switchCharacterLayout(char: string): string {
   if (en_ru.has(char)) {
     // console.log(char + " detected in en dict")
-    switchLayout("Russian");
```

Comment:
I believe here should be something like `switchLayout(layoutRu.value);`

Reply:
This is a duplicate switch, the main one is already in line 34.

Reply:
Good call